src/common/tiles/PersonTile/styled.js

```diff
@@ -9,71 +9,66 @@ export const Wrapper = styled.div`
   margin-top: 24px;
   justify-content: center;
   background-color: ${({ theme }) => theme.colors.lightGrey};
-`;
+`; */
 
 export const Container = styled.div`
-  width: 208px;
-  height: 339px;
+  padding: 16px;
+  word-break: break-word;
+  width: 100%;
+  height: 100%;
   background-color: ${({ theme }) => theme.colors.white};
   border-radius: ${({ theme }) => theme.radius.small};
-  padding: 16px;
+  box-shadow: ${({ theme }) => theme.shadow.mainShadow};
   display: flex;
   flex-direction: column;
-  box-shadow: ${({ theme }) => theme.shadow.mainShadow};
-
-  @media (max-width: ${({ theme }) => theme.breakpoints.mobileMax}) {
-    width: 136px;
-    height: 245px;
+  
+  @media (max-width:${({ theme }) => theme.breakpoints.large}) {
     padding: 8px;
   }
 `;
 
 export const ProfileImage = styled.img`
-  width: 177px;
-  height: 232.31px;
-  display: block;
-  flex-shrink: 0;
+  width: 100%;
   border-radius: ${({ theme }) => theme.radius.small};
-  object-position: top;
-  object-fit: cover;
-
-  @media (max-width: ${({ theme }) => theme.breakpoints.mobileMax}) {
-    width: 120px;
-    height: 178px;
-  }
+  aspect-ratio: 2/3;
 `;
 
 export const Name = styled.p`
   font-size: 22px;
-  font-weight: 500;
+  font-weight: 500px;
+  margin: 12px 0px 0px 0px;
+  line-height: 1.3;
   color: ${({ theme }) => theme.colors.black};
-  text-align: center;
-  margin-bottom: 0;
-  line-height: 28.6px;
+  flex: 1 1 auto;
+	text-align: center;
+	
+  @media (max-width:${({ theme }) => theme.breakpoints.large}) {
+    font-size: 16px;
+    margin: 8px 0px 0px 0px;
+  }
 
-  @media (max-width: ${({ theme }) => theme.breakpoints.mobileMax}) {
-    font-size: 14px;
-    line-height: 18.2px;
+  @media (max-width:${({ theme }) => theme.breakpoints.small}) {
+		font-size: 14px;
   }
 `;
 
-export const Character = styled.h2`
+export const Info = styled.p`
   font-weight: 400;
-  font-size: 18px;
-  line-height: 150%;
-  display: grid;
-  grid-template-columns: auto;
-  justify-content: center;
-  text-align: center;
-  margin-top: 8px;
-  color: ${({ theme }) => theme.colors.darkGrey};
+  display: none;
 
-  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
-    font-size: 15px;
-    line-height: 140%;
-  }
-  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+  ${({ big }) => big && css`
+    font-size: 18px;
+    font-weight: 400;
+		line-height: 1.3;
+		color: ${({ theme }) => theme.colors.darkGrey};
+		margin: 8px 0px 0px 0px;	
+    display: block;
+		flex: 1 1 auto;
+		text-align: center;
+  `}
```

Comment:
Nie jest to przytyk do ciebie bynajmniej, ale zauwazylem, ze dziwne jest tutaj formatowanie na githubie a w vsc jest normalne, takze pisze jako ciekawostke bo wczesniej sie z tym chyba nie spotkalem :)

Reply:
A co masz na myśli? Bo chyba nie kumam o czym mówisz.